image/nginx/nginx.conf

```diff
@@ -0,0 +1,100 @@
+user www-data;
```

Comment:
You previously had this set as root, noting a bug. Does that bug still exist? I had it working with `www-data` as the user.